app/assets/stylesheets/_forms.scss

```diff
@@ -1,3 +1,15 @@
+.elements-error-summary {
+  border: 5px solid #b10e1e;
```

Comment:
I'm not sure if I understand. I haven't found any colour variables in `govuk_admin_template` which makes me wonder that is safe to use the colours provided by `govuk_frontend_toolkit ` since you are using the gem already.

I think that for `govuk_admin_template` to use the colours from `govuk_frontend_toolkit` it would require extracting that into another gem that would have common attributes between those two and then make it a requirement into both. Then yes, there's 1 source of truth for any commonalities.

I would prefer reusing something that has been already defined, decisions change all the time, maybe today red is `#b10e1e` but tomorrow is `#ff0e1e` and it's just harder to find and change many files than just one.